restli-server/src/main/java/com/linkedin/restli/internal/server/ServerResourceContext.java

```diff
@@ -58,6 +58,17 @@
    */
   String CONTEXT_PROJECTION_MASKS_KEY = ServerResourceContext.class.getName() + ".projectionMasks";
 
+  /**
+   * Local attribute key to indicate that this request will be served by an in-process Rest.li server. Value must
+   * be a {@link Boolean}.
+   */
+  String CONTEXT_IN_PROCESS_RESOLUTION_KEY = ServerResourceContext.class.getName() + ".inProcessResolution";
```

Comment:
Where did you set these two newly added context key?

Reply:
I will set it in the clients when making the request. For example in RemoteAndLocalRestClient in our internal GQL repo.